tern/classes/docker_image.py

```diff
@@ -130,11 +130,17 @@ def set_layer_created_by(self):
                 else:
                     self._layers[index].created_by = ''
                 index = index + 1
+                if index is self.until_layer:
+                    break
 
-    def load_image(self):
+    def load_image(self, until_layer=0):
         """Load metadata from an extracted docker image. This assumes the
         image has already been downloaded and extracted into the working
         directory"""
+        if until_layer is None:
+            until_layer = 0
+        if until_layer != 0:
+            super(DockerImage, type(self)).until_layer.fset(self, until_layer)
```

Comment:
Why use `super()` here (and below) versus directly assigning/accessing the property value like we do with the other properties for the image below? i.e. `self._until_layer = until_layer`